services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_minis.jsx

```diff
@@ -202,17 +217,18 @@ export default class UnitTemplateMinis extends React.Component {
         value: `${baseLink}${qs}`
       }
     });
-    typeOptionsForDropdown.unshift({ label: 'All packs', value: '/activities/packs' });
+    typeOptionsForDropdown.unshift({ label: 'All Packs', value: '/activities/packs' });
 
     const allPacksLink = (
       <Link
         className={!selectedTypeId ? 'active' : null}
         to={`${baseLink}${this.generateQueryString(currentCategory, data.selectedGradeLevel)}`}
-      >All packs</Link>
+      >All Packs</Link>
```

Comment:
These capitalization changes make sense given that other pieces of text on the page (ie "Reading Texts") are capitalized, but I thought we were moving towards sentence-case sitewide. It might be worth confirming with Jack that we do want this page to be an exception to that.

Reply:
so I checked in with Jack who checked in with Peter, and we do want the title casing in this context